client/blocks/eligibility-warnings/index.jsx

```diff
@@ -87,7 +87,7 @@ const EligibilityWarnings = props => {
 
 	const classes = classNames( {
 		'eligibility-warnings__message': true,
-		'is-placeholder': isPlaceholder,
```

Comment:
It could have been fine to leave this as a modifier class if that's what it truly is and simply update the style selectors to be more specific: `.eligibility-warnings__message.is-placeholder`